### PR TITLE
extended sections with evaluation metrics and added links to blog posts

### PR DESCRIPTION
**Pull request by jakubczakon**

### CODE REVIEW

File: model_selection/imbalanced/imbalanced_metrics.ipynb [hunk 1]
```diff
@@ -1300,7 +1300,16 @@
...
+    "With this knowledge in mind, let's turn our heads back to F1 score's calculation. Precision and recall both have true positives in the numerator, but they have different denominators. In order for an average to be valid, we need the values to be in the same scaled units. In our traveling distance example miles per hour need to be compared over the same number of hours. Thus, to take the average of precision and recall really only makes sense to average their reciprocals (adjusts for scale), thus the harmonic mean.\n",
+    "\n",
+    "If the business case warrants it, you may want to put more weight on recall than precision or vice-versa. In that case a more general version of the F score called F beta score could be useful. \n",
+    "\n",
+    "\\begin{align}\n",
+    "Fbeta &= (1+\\beta^2) * \\frac{\\text{precision} * \\text{recall}}{\\beta^2*\\text{precision} + \\text{recall}}\n",
+    "\\end{align}\n",
+    "\n",
+    "With beta>1 you focus more on recall, with 0<beta<1 you put more weight on precision.\n",
```
Comment (ethen8181) [inline model_selection/imbalanced/imbalanced_metrics.ipynb]: Change the beta to the latex way, $beta$ to make it consistent with the formula above perhaps?

File: model_selection/imbalanced/imbalanced_metrics.ipynb [hunk 2]
```diff
@@ -1300,7 +1300,16 @@
...
+    "With this knowledge in mind, let's turn our heads back to F1 score's calculation. Precision and recall both have true positives in the numerator, but they have different denominators. In order for an average to be valid, we need the values to be in the same scaled units. In our traveling distance example miles per hour need to be compared over the same number of hours. Thus, to take the average of precision and recall really only makes sense to average their reciprocals (adjusts for scale), thus the harmonic mean.\n",
+    "\n",
+    "If the business case warrants it, you may want to put more weight on recall than precision or vice-versa. In that case a more general version of the F score called F beta score could be useful. \n",
```
Comment (ethen8181) [inline model_selection/imbalanced/imbalanced_metrics.ipynb]: More of a nitpick. Perhaps change the "you" to "we". e.g.

- we may want to put more weight.
- We focus more on recall. etc.

**Review by ethen8181 — COMMENTED** [review]
I left some minor comments. And if you like, feel free to add yourself to the contributor list (contributors.md).

File: model_selection/imbalanced/imbalanced_metrics.ipynb (hunk 1 above)
Comment (jakubczakon) [reply model_selection/imbalanced/imbalanced_metrics.ipynb]: On it.

File: model_selection/imbalanced/imbalanced_metrics.ipynb (hunk 2 above)
Comment (jakubczakon) [reply model_selection/imbalanced/imbalanced_metrics.ipynb]: You are right, I'll change that.

Comment (jakubczakon) [reply model_selection/imbalanced/imbalanced_metrics.ipynb]: Done @ethen8181

File: model_selection/imbalanced/imbalanced_metrics.ipynb (hunk 1 above)
Comment (jakubczakon) [reply model_selection/imbalanced/imbalanced_metrics.ipynb]: Done @ethen8181

Comment (ethen8181) [reply model_selection/imbalanced/imbalanced_metrics.ipynb]: I was also referring to the paragraph With beta>1 you focus more on recall, with 0<beta<1 you put more weight on precision.

The beta for `beta > 1`, and `0<beta<1`, should ideally be in latex format to be consistent with your formula.

Comment (jakubczakon) [reply model_selection/imbalanced/imbalanced_metrics.ipynb]: Done.

**Comment by ethen8181** [comment]
Thanks!!! Merged.

**Pull request merged**